docs/dev-setup.rst

```diff
@@ -90,7 +96,21 @@ To setup your local environment you should create a virtualenv and install the
 necessary requirements::
 
     mkvirtualenv --python=/usr/bin/python3.4 serviceinfo
```

Comment:
Apparently I was wrong. `virtualenv` seems to find pythonX.Y anywhere on the path, but `mkvirtualenv` for some reason tries to find it without using the path and fails.

What we've used elsewhere is

    mkvirtualenv -p `which python3.4` serviceinfo

which should work regardless.  How does that sound?